src/utils/copyText.ts

```diff
@@ -0,0 +1,7 @@
+export function copyText(text: string) {
```

Comment:
This file is not needed to be at the root utils, place it near the component please